lib/matplotlib/axis.py

```diff
@@ -2020,7 +2020,10 @@ def get_tick_space(self):
         # There is a heuristic here that the aspect ratio of tick text
         # is no more than 3:1
         size = tick.label1.get_size() * 3
-        return int(np.floor(length / size))
+        if size > 0:
+            return int(np.floor(length / size))
+        else:
```

Comment:
It is probably a bit of apples and oranges since I find the way I wrote it clearest. I looked around for something in numpy that would cast infinities to integers better, but I didn't come up with anything.